cmd/promlts/main.go

```diff
@@ -54,9 +55,9 @@ func main() {
 		}
 		logger = log.NewLogfmtLogger(os.Stderr)
 		logger = log.NewSyncLogger(logger)
-		logger = level.NewFilter(logger, lvl)
 		logger = log.With(logger, "ts", log.DefaultTimestampUTC)
 		logger = log.With(logger, "caller", log.DefaultCaller)
+		logger = level.NewFilter(logger, lvl)
```

Comment:
wow, you are changing that around a lot (:

Reply:
Not sure how that flipped back – before was correct :)